Survey/Sources/Supports/Extensions/Foundation/Date+.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Date+.swift
```

Comment:
Normally we will add the functionality description to extension file name such as `Date+StringFormat`, this way it makes the file more readable. If you decided to add multiple functions to one extension file, I suggest using `Date+Extensions` so that this file is easier to understand what it is. Since there are multiple files with bare `+`, this is optional. 😄

Reply:
Thanks, It looks easier to understand what it is when using your suggestion 😄 , I renamed it at https://github.com/nkhanh44/ic-survey-ios/pull/18/commits/741e27bb29df17a3f373e2c4f4b8fa67ba11a3cd